lib/Test.rakumod

```diff
@@ -1,3 +1,4 @@
+use v6.c;
```

Comment:
Any reason why you're doing this? It makes sense, but it's unrelated to the other changes.

Reply:
I don't want a separate commit, let alone a PR, for just one line.